tsconfig.settings.json

```diff
@@ -1,5 +1,6 @@
 {
     "compilerOptions": {
+      "skipLibCheck": true,
```

Comment:
@wildmolasses Curious if you have any better solutions here, since I know you're not a fan of `skipLibCheck`. The problem was that `@types/mocha` and `@types/jest` have conflicting definitions, and since we need both running `yarn build` in the `umbra-js` package failed with the errors described here: https://stackoverflow.com/questions/55680391/typescript-error-ts2403-subsequent-variable-declarations-must-have-the-same-typ

Removing one of the two packages doesn't seem like a great option since we use Jest in the frontend and Mocha in the contracts/umbra-js packages

Reply:
[`skipLibCheck`](https://www.testim.io/blog/typescript-skiplibcheck/) is a code smell. we want to know about declaration errors whether they're in library code or our code, and resolve them where possible.

furthermore, codebases that consume a package using `skipLibCheck` must also use `skipLibCheck`

If possible, let's avoid `skipLibCheck`; here, we do it by separating the mocha and jest types via a nohoist declaration so that each subpackage has the relevant declaration file.